demo/README.md

```diff
@@ -21,3 +23,19 @@ different handlers.
     This will deploy the app to your default project, which you can configure using gcloud:
 
            gcloud config set project <project-id>
+
+## Shell
+
+1. First, ensure your environment is setup.
+
+           export DATASTORE_PROJECT_ID=<project-id>
+           export DATASTORE_USE_PROJECT_ID_AS_APP_ID=True
```

Comment:
Does "True" need to be capitalized? If not, I think other docs don't do it. If so, not a big deal.